### DISCUSSION TITLE
Porte e utenze

### DISCUSSION THREAD

**Discussion by marcoronk** [globaleaks/globaleaks-whistleblowing-software#3642] [Q&A]
Buongiorno,
è possibile cambiare la porta di default 443 con un'altra più alta e far girare l'applicazione in docker con un utente diverso da root?
Grazie.

Good morning,
is it possible to change the default port 443 with an higher port, and run the application in docker with a different user than root?
Thanks

Marco

**Comment by marcoronk** [comment]
Uno dei problemi l'ho risolto correggendo questo file:

https://github.com/globaleaks/globaleaks-docker-scripts/blob/main/data/supervisord.conf

in questo file di configurazione globaleaks viene chiamato con il flag --user ma mi va in errore. Nello scriptdi avvio globaleaks che ho nel container, chiamandolo con -v non mostra questo flag.

**Comment by evilaliv3** [comment]
Buonasera @marcoronk 

Stiamo di fatto lavorando a correggere lo script docker e a implementare pieno rootless.

Se vuoi puoi raggiungerci sul nostro canale community dove ne stiamo discutendo: community.globaleaks.org

**Comment by marcoronk** [comment]
Grazie, ho visto che avete rilasciato un nuovo aggiornamento. Ma mi sa che c'è un refuso nel dockerfile o sbaglio io? vedo un:

EXPOSE 4443

con netstat vedo che apre alla 8443. 
Grazie.

**Comment by evilaliv3** [comment]
Grazie per averlo notato. si è certamente un refuso.

Stiamo ultimando una configurazione Docker  rootless. Aveste suggerimenti sono benvenuti.

**Reply by marcoronk** [reply]
Grazie per la risposta. Stiamo cercando di installarlo su un ambiente openShift / Kubernetes ma ci danno problemi le redirect. 

Nel caso in cui acceda al container via IP mi da una redirect (http 301) verso https://localhost/
Nel caso acceda tramite hostname mi riotrna direttamente bad request (http 400)

Credo siano le redirect fatte con IpTables dentro lo script di avvio nel blocco Tor Sandboxing. Sto cercando di disabilitare questo blocco per capire cosa non va. 
Grazie.
